app/assets/stylesheets/custom.scss

```diff
@@ -257,3 +262,22 @@ ul, menu, dir {
   margin-top: 100px;
```

Comment:
Properties should be ordered display, list-style-type, -webkit-margin-after, -webkit-margin-before, -webkit-margin-end, -webkit-margin-start, -webkit-padding-start<br>`0px` should be written without units as `0`